pages/docs/doc/ibge.json

```
@@ -217,6 +217,123 @@
                     }
                 }
             }
+        },
+        "/ibge/nomes/v1/{nome}": {
+            "get": {
+                "tags": ["IBGE"],
+                "summary": "Busca a frequencia de um determinado nome",
+                "description": "Um ou mais nomes delimitados pelo caracter | (pipe)",
```

```suggestion
                "description": "Um ou mais nomes delimitados pelo caractere | (pipe)",
```

Será que em vez de receber vários nomes assim, não seria mais interessante receber como `/ibge/nomes/v1/?nomes[]=Ana&nomes[]=Pedro` por ser mais comum?